packages/@cdktf/cli-core/src/lib/cdktf-stack.ts

```
@@ -273,8 +273,10 @@ export class CdktfStack {
         stackName: this.stack.name,
         error: String(e),
       });
+      throw e;
```

We swallowed an error here by not rethrowing